packages/apps/testbench-app/src/components/SpaceToolbar.tsx

```diff
@@ -51,9 +50,7 @@ export const SpaceToolbar = ({
 
   return (
     <Toolbar.Root classNames='p-1'>
-      <Toolbar.Button title='Create space.' onClick={() => onCreate()}>
-        <Plus />
-      </Toolbar.Button>
+      <IconButton icon='ph--plus--regular' title='Create space.' onClick={() => onCreate()} />
```

Comment:
Because of Toolbar’s special keyboard access, this should be:

```suggestion
      <Toolbar.IconButton icon='ph--plus--regular' title='Create space.' onClick={() => onCreate()} />
```